drivemount/drivemount.c

```diff
@@ -36,78 +36,78 @@ static const char drivemount_iid[] = "DriveMountApplet";
 static const char factory_iid[] = "DriveMountAppletFactory";
 
 static void
-change_orient (MatePanelApplet *applet, MatePanelAppletOrient o, DriveList *drive_list)
+change_orient (MatePanelApplet *applet,
+               MatePanelAppletOrient o,
+               DriveList *drive_list)
 {
     GtkOrientation orientation;
 
     switch (o) {
-    case MATE_PANEL_APPLET_ORIENT_LEFT:
-    case MATE_PANEL_APPLET_ORIENT_RIGHT:
-	orientation = GTK_ORIENTATION_VERTICAL;
-	break;
-    case MATE_PANEL_APPLET_ORIENT_UP:
-    case MATE_PANEL_APPLET_ORIENT_DOWN:
-    default:
-	orientation = GTK_ORIENTATION_HORIZONTAL;
-	break;
+        case MATE_PANEL_APPLET_ORIENT_LEFT:
+        case MATE_PANEL_APPLET_ORIENT_RIGHT:
+            orientation = GTK_ORIENTATION_VERTICAL;
+            break;
+        case MATE_PANEL_APPLET_ORIENT_UP:
+        case MATE_PANEL_APPLET_ORIENT_DOWN:
+        default:
+            orientation = GTK_ORIENTATION_HORIZONTAL;
     }
     drive_list_set_orientation (drive_list, orientation);
 }
 
 static void
-size_allocate (MatePanelApplet  *applet,
-	       GdkRectangle *allocation,
-	       DriveList    *drive_list)
+size_allocate (MatePanelApplet *applet,
+               GdkRectangle    *allocation,
+               DriveList       *drive_list)
 {
     int size;
 
     switch (mate_panel_applet_get_orient (applet)) {
-    case MATE_PANEL_APPLET_ORIENT_LEFT:
-    case MATE_PANEL_APPLET_ORIENT_RIGHT:
-	size = allocation->width;
-	break;
-    case MATE_PANEL_APPLET_ORIENT_UP:
-    case MATE_PANEL_APPLET_ORIENT_DOWN:
-    default:
-	size = allocation->height;
-	break;
+        case MATE_PANEL_APPLET_ORIENT_LEFT:
+        case MATE_PANEL_APPLET_ORIENT_RIGHT:
+            size = allocation->width;
+            break;
+        case MATE_PANEL_APPLET_ORIENT_UP:
+        case MATE_PANEL_APPLET_ORIENT_DOWN:
+        default:
+            size = allocation->height;
     }
     drive_list_set_panel_size (drive_list, size);
 }
 
 static void
 change_background (MatePanelApplet               *applet,
-		   MatePanelAppletBackgroundType  type,
-		   GdkColor                  *colour,
-		   cairo_pattern_t           *pattern,
-		   DriveList                 *drivelist)
+                   MatePanelAppletBackgroundType  type,
+                   GdkColor                      *colour,
+                   cairo_pattern_t               *pattern,
+                   DriveList                     *drivelist)
 {
     switch (type) {
-    case PANEL_NO_BACKGROUND:
-	drive_list_set_transparent (drivelist, FALSE);
-	break;
-
-    case PANEL_COLOR_BACKGROUND:
-    case PANEL_PIXMAP_BACKGROUND:
-	drive_list_set_transparent (drivelist, TRUE);
-	break;
+        case PANEL_NO_BACKGROUND:
+            drive_list_set_transparent (drivelist, FALSE);
+            break;
+
+        case PANEL_COLOR_BACKGROUND:
+        case PANEL_PIXMAP_BACKGROUND:
+            drive_list_set_transparent (drivelist, TRUE);
+            break;
```

Comment:
this break sentence does nothing

Reply:
Because of the empty line 89?
Should i remove this line?

Reply:
Opps, got it.
You meant the break in line 93.
I will  drop it.